test/fixtures/tape-standalone.js

```diff
@@ -9,10 +9,51 @@ if (navigator.userAgent.includes("Chrome/")) {
   browser = "Firefox";
 }
 
+function getArgs(/* desc, options, fn */) {
+  let desc = "(anonymous)";
+  // By setting `timeout` to `0` here, we are allowing V8 to consider
+  // it a to have a signature of `{timeout: number}` instead of `{}`
+  // with a `timeout` property, which is easier on the C++ portion.
+  let options = {timeout: 0};
+  /** @type {function(import("tape").Test):void|Promise<void>} */
+  let fn;
+
+  for (let i = 0, arg; i < arguments.length; i++) {
+    arg = arguments[i];
+    switch (typeof arg) {
+      case "string":
+        desc = arg;
+        break;
+      case "function":
+        fn = arg;
+        break;
+      case "object":
+        Object.assign(options, arg);
+        break;
+    }
+  }
+
+  if (!options.timeout) {
+    options.timeout = DEFAULT_TIMEOUT;
+  }
+
+  return {
+    desc,
+    options,
+    fn,
+  };
+}
+
 // Export as a global a wrapped test function which enforces a timeout by default.
-window.test = (desc, fn) => {
-  tape(`${desc} (${browser})`, async (t) => {
-    t.timeoutAfter(DEFAULT_TIMEOUT);
+// eslint-disable-next-line
+/**
+ * @param {string} [desc]
+ * @param {object} [options]
+ * @param {(test: import("tape").Test) => void|Promise<void>} fn
```

Comment:
Let's remove this JSDoc. The names are self-explanatory, and "documentation" is available in the form of existing examples (tests). Anyone who cares can look in this file and see that this is simply a wrapper around `tape`.

Reply:
This&nbsp;is there for&nbsp;intellisense in&nbsp;IDEs.